releases/release-1.13/README.md

```diff
@@ -0,0 +1,139 @@
+# Kubernetes 1.13 Release Schedule
+
+*Handy Links*
+
+* [This document](https://git.k8s.io/sig-release/releases/release-1.13/README.md)
+* [Release Team](https://git.k8s.io/sig-release/releases/release-1.13/release_team.md)
+* Zoom: join Google Group to receive meeting invite
+* [Slack](https://kubernetes.slack.com/messages/sig-release/)
+* [Forum](https://groups.google.com/forum/#!forum/kubernetes-sig-release)
+* [Feature Tracking Sheet](http://bit.ly/k8s113-features)
+* [Targeting Issues and PRs to This Milestone](https://git.k8s.io/community/contributors/devel/release.md)
+* [Meeting Minutes](http://bit.ly/k8s113-minutes)
+* [CI Signal Report](http://bit.ly/k8s113-cisignal)
+* [Retrospective Document](http://bit.ly/k8s113-retro)
+
+**tl;dr** The 1.13 release cycle begins on **Monday, October 1st, 2018**, and ends on release day, **Thursday, December 06, 2018**.  Feature freeze is **Tuesday, October 23, 2018**.  Code slush is **Friday, November 09, 2018**. Code freeze begins **Friday, November 16, 2018** and ends **Monday, December 03, 2018**, but [may vary depending on CI Signal](#conditionally-shortened-code-freeze). Docs must be completed and reviewed by **Friday, November 30, 2018**. Release blocking test case additions must be completed by in conjunction with features by code freeze.
```

Comment:
Maybe I'm getting cross-eyed between the table and the paragraph but the table shows as the release day as Jan 6, right?

Reply:
Ah, found it -- the dates from code freeze upto `1.13.0-rc.1 released from branch` should be in Nov instead of Dec.

And the dates in Jan should be Dec. :)